monitor/grafana/zeebe.json

```diff
@@ -8749,7 +8749,7 @@
             "alertThreshold": true
           },
           "percentage": false,
-          "pluginVersion": "9.5.2",
+          "pluginVersion": "9.3.6",
```

Comment:
💭 Weird, all other olugins were upgrade but this one downgraded? 🤷‍♂️

Reply:
Sorry I did a git add -p and probably added this wrongly.